cypress/integration/example_spec.js

```diff
@@ -157,8 +157,11 @@ describe('pipe()', () => {
     })
 
     context('when visiting a page', () => {
+      /** @param $el {JQuery} */
       const getFirst = $el => $el.find('#first')
+      /** @param $el {JQuery} */
       const getSecond = $el => $el.find('#second')
+      /** @param $el {JQuery} */
```

Comment:
These JSDoc types are checked. Perhaps there should be more official Typescript type checking. Right now I manually verify in my editor. There are other type errors I'd have to cast as `any`, but this is a JS file without a TS loader...

Perhaps I should look into this more if others want to contribute more to the source code. Right now it is a simple enough library and it isn't very difficult to maintain.